client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -58,14 +58,14 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
 
   pythonLoading = false;
   webApplicationFrameworks: DropDownElement<string>[] = [
-    {
-      displayLabel: 'ASP.NET',
-      value: WebAppFramework.AspNetWap,
-    },
     {
       displayLabel: 'ASP.NET Core',
       value: WebAppFramework.AspNetCore,
     },
+    {
+      displayLabel: 'ASP.NET',
```

Comment:
I think you will also need to change default selected value in line no 89

Reply:
Done